mandelbrot-set-python/README.md

```diff
@@ -0,0 +1,26 @@
+# Draw the Mandelbrot Set in Python
+
+Code snippets and Python scripts supplementing the [Draw the Mandelbrot Set in Python](https://realpython.com/mandelbrot-set-python) tutorial on [Real Python](https://realpython.com/).
+
+## Installation
+
+This code requires the following libraries:
+
+- Matplotlib
+- NumPy
+- Pillow
+- SciPy
```

Comment:
Since you're listing these explicitly, I think it would be more helpful to use the exact PyPI names (so basically change to lowercase) as this would allow any power reader to install manually instead of using the prepared and locked `requirements.txt`:

```suggestion
- `matplotlib`
- `numpy`
- `pillow`
- `scipy`
```

Reply:
These are the proper names of the libraries I'm referring to. PyPI is case insensitive, so it will accept uppercase, lowercase, and everything in between.